server/server.js

```diff
@@ -17,5 +17,49 @@ app.get('/reviews', (req, res) => {
     .then(result => res.end(JSON.stringify(result)));
 });
 
+// GET /reviews/meta
+app.get('/reviews/meta', (req, res) => {
+  const {
+    product_id,
+  } = req.query;
+
+  // Run the db query
+});
+
+// POST /reviews
+app.post('/reviews', (req, res) => {
+  const {
+    product_id,
```

Comment:
Very clean and understandable way of passing the properties from the request object.